cli/cli_test.go

```diff
@@ -713,13 +713,6 @@ func TestCanListAppsWithCountAndFilter(t *testing.T) {
 	appsServiceMock.AssertExpectations(t)
 }
 
-func TestCanPublishAnApp(t *testing.T) {
```

Comment:
why removing it? (I guess it failed because you removed the ability to publish an app with the default manifest file?)